pkg/operator/controller/ingress/controller.go

```diff
@@ -97,17 +97,19 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err := c.Watch(&source.Kind{Type: &operatorv1.IngressController{}}, &handler.EnqueueRequestForObject{}); err != nil {
+	scheme := mgr.GetClient().Scheme()
+	mapper := mgr.GetClient().RESTMapper()
```

Comment:
Why not create a variable for the cache too? I saw that in some of the files you had to modify there was `operatorCache` variable. Same can be done here and in other files. It's not a big deal but potentially it can win us some time spent on the function calls.

Reply:
Done, cleaned up variables to use `operatorCache` pattern.

Reply:
And noticed a mistake in `pkg/operator/controller/route-metrics/controller.go` where I should have used `newCache` while doing this.

Reply:
Actually, I take that back after reviewing the changes to `pkg/operator/controller/route-metrics/controller.go`, it was correct the way I had it originally.